CHANGELOG.rst

```diff
@@ -14,6 +14,8 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+* Log error on waffle object creation when name includes a blank space as a prefix or suffix.
```

Comment:
Do you want to give this a version and bump the version, or do you just want to this to wait? Either way. Thanks.

Reply:
Let's keep it that way for now, I prefer to make release commits separately.